File: auto-docs/executables/r/stacked_bar.r
library(plotly)

p <- plotly(username='theengineear', key='o9zlr0hy6z')

trace1 <- list(
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(20, 14, 23), 
  name = "SF Zoo", 
  type = "bar"
)
trace2 <- list(
  x = c("giraffes", "orangutans", "monkeys"), 
  y = c(12, 18, 29), 
  name = "LA Zoo", 
  type = "bar"
)
data <- list(trace1, trace2)
layout <- list(
  xaxis = list(type = "category"), 
  categories = c("giraffes", "orangutans", "monkeys"), 
  barmode = "stack"
)

response <- p$plotly(data, kwargs=list(layout=layout, filename="stacked-bar", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/matlab/stacked_bar.m
signin('theengineear', 'o9zlr0hy6z')

trace1 = struct(...
  'x', { {'giraffes', 'orangutans', 'monkeys'} }, ...
  'y', [20, 14, 23], ...
  'name', 'SF Zoo', ...
  'type', 'bar');
trace2 = struct(...
  'x', { {'giraffes', 'orangutans', 'monkeys'} }, ...
  'y', [12, 18, 29], ...
  'name', 'LA Zoo', ...
  'type', 'bar');
data = {trace1, trace2};
layout = struct(...
    'xaxis', struct('type', 'category'), ...
    'categories', { {'giraffes', 'orangutans', 'monkeys'} }, ...
    'barmode', 'stack');

response = plotly(data, struct('layout', layout, 'filename', 'stacked-bar', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url